### PR TITLE
Various Fixes and Add HoloCom System.

### PR DESCRIPTION
**Pull request by givemedeath**

### CODE REVIEW

File: SWLOR.Game.Server/Conversation/HoloCom.cs [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+
+        public void LoadMainMenu()
+        {            
+            ClearPageResponses("MainPage");
+
+            string header = ColorTokenService.Green("HoloCom Menu\n\n");
+            SetPageHeader("MainPage", header);
+
+            NWPlayer user = GetPC();
+            Console.WriteLine("LoadMainMenu: user = " + user.Name + ", receiver = " + HoloComService.GetCallReceiver(user).Name);
```
Comment (zunath) [inline SWLOR.Game.Server/Conversation/HoloCom.cs]: Can you swap these to LogginService calls or remove them? Server's gonna get spammed. If you need them for testing that's ok too. Just need to do something about it before we go live with it.

File: SWLOR.Game.Server/Service/HoloComService.cs [hunk 2]
```diff
@@ -0,0 +1,330 @@
...
+            {
+                ActionPlayAnimation(animation);
+            });
+
+            receiver.DelayAssignCommand(() =>
+            {
+                ActionSpeakString(text, talkvolume);
+            },0.3f);
+
+            /* animation stuff
```
Comment (zunath) [inline SWLOR.Game.Server/Service/HoloComService.cs]: Probably want to leave this debug info out of the code base.

Comment (givemedeath) [reply SWLOR.Game.Server/Service/HoloComService.cs]: I left it there so the future change and potential solution is documented somewhere. Should move it to an "Issue" instead?

File: SWLOR.Game.Server/Conversation/HoloCom.cs (hunk 1 above)
Comment (givemedeath) [reply SWLOR.Game.Server/Conversation/HoloCom.cs]: Yeah, i thought i removed that already. Woops

File: SWLOR.Game.Server/Service/HoloComService.cs (hunk 2 above)
Comment (zunath) [reply SWLOR.Game.Server/Service/HoloComService.cs]: Yea let's move it to an issue and just tag the PR so we can reference it in the future. I just think it confuses things being in the code itself

**Pull request merged**